sw/airborne/boards/chimera/chibios/v1.0/chimera.h

```diff
@@ -142,7 +142,7 @@
 
 // AUX1
 #if USE_ADC_2
-#define AD1_2_CHANNEL ADC_CHANNEL_IN2
+#define AD1_2_CHANNEL ADC_CHANNEL_IN3
```

Comment:
I think this came from `git merge master` - should I rebase or merge-rebase again?